_adr/81/index.adoc

```diff
@@ -26,3 +26,152 @@ Excluding storage sizing, the Kafka service currently supports a single size of
+
+## Current situation
+// Where are we now?
+
+The service does not offer any scaling options, except for storage, which is orthogonal to this problem.
+
+## Proposal
+// What is the decision being proposed
+
+### Conceptual model
+Conceptually we define a _series_, which is an ordered list of _sizes_ of instance. The ordering of the list MUST be defined by the ordering of the functional attributes that define the service. For example, for the example series "ex1", we might have the sizes:
+
+`ex1.small`:: Ingress 30MB/s, egress 30MB/s, 1000 partitions, 500 connections per instance, max 100 connections per second, 1MB max message size.
+`ex1.medium`:: Ingress 60MB/s, egress 60MB/s, 2000 partitions, 1000 connections per instance, max 100 connections per second, 1MB max message size.
+`ex1.large`:: Ingress 90MB/s, egress 90MB/s, 3000 partitions, 1500 connections per instance, max 100 connections per second, 1MB max message size.
+
+In this example most of the functional attributes scales linearly, with the exception of connection rate and max message size. In general, linearity SHALL NOT be a requirement, even for those attributes which do scale up.
+For example, it would have been acceptable to define the `ex1.large` ingress as 100M/s, even though this is not 3× the `ex1.small` ingress, even when the other `ex1.large` functional parameters are 3× their `ex1.small` values.
+
+### UI/Fleet manager interface
+
+This ADR doesn't mandate any particular presentation for how the scale options are presented in the UI.
+In particular, while there is only one series defined, there is no requirement to display series in the UI at all.
+
+The fleet manager SHALL be the source of truth for the series and sizes available, and how those depend on cloud and region. 
+The fleet manager SHALL provide an endpoint which the UI uses to determine the available options.
+The UI MUST NOT make assumptions about linearity of scaling, instead it uses only the series and sizes provided by the endpoint.
+This endpoint will return a map of lists.
+The map keys will be the series, and the elements of the list will be the available sizes within that series, in increasing order. 
+Nothing is implied by the order of the keys in the series map, e.g. an "ex2" may occur before or after an "ex1" and the order doesn't imply anything about those series.
+
+Reusing the example series defined above:
+[source,json]
+----
+"sizes": {
+  "ex1": [
+    { 
+      "name": "small",
+      "parameters": {
+        "ingress": "30MB/s", // should this really be a map, or a list of parameters, thus allowing more detailed descriptions in addition to a bare name+value?
+        "egress": "30MB/s",
+        "partitions": "1000",
+        "connections": "500",
+        "connectionRate": "100",
+        "maxMessageSize": "1MB"
+      }
+    },
+    { 
+      "name": "medium",
+      "parameters": {
+        "ingress": "60MB/s",
```

Comment:
@gunnarmorling amazing point! I think if that would be possible it will be good to have it separated.
It will be more future proof as UI might use different shortcuts/full names of the units.

That will be also good pattern for validation purposes (although we only use partitions value for validation now)

Reference for UI:
https://github.com/bf2fc6cc711aee1a0c2a/kafka-ui/blob/main/src/constant/constants.ts#L19


https://github.com/bf2fc6cc711aee1a0c2a/kafka-ui/blob/main/src/constant/constants.ts#L19

Reply:
Yes please keeping them separate would also allow for easier internationalization of the UoM (not saying we'll have to do that, I have no idea if MB/s can be written differently in other languages, but all the above pros still apply)